apps/package.json

```diff
@@ -15,14 +15,15 @@
     "test:integration": "grunt integrationTest",
     "test:entry": "grunt karma:entry",
     "clean": "grunt clean",
-    "build": "MOOC_DEV=1 grunt build; grunt logBuildTimes",
-    "build:dist": "NODE_ENV=production MOOC_DEV=0 grunt clean build; grunt logBuildTimes",
+    "build": "MOOC_DEV=1 grunt build; npm run compile-firebase-rules; grunt logBuildTimes",
+    "build:dist": "NODE_ENV=production MOOC_DEV=0 grunt clean build; npm run compile-firebase-rules; grunt logBuildTimes",
```

Comment:
Do you want `compile-firebase-rules` to run every time here, or only when the preceding build step succeeds? (That is, should this be `;` or `&&`?)

Alternatively, you could encode these as separate tasks `postbuild` and `postbuild:dist` (which npm will automatically run after the respective tasks).  Don't worry about ordering with the `logBuildTimes` task - it's only gathering data from the grunt process either way.